src/_posts/platform/databases/2000-01-01-access.md

```diff
@@ -170,6 +170,14 @@ ssh -L 10000:my-db.postgresql.a.osc-fr1.scalingo-dbs.com:30000 git@ssh.osc-fr1.s
 Then you connect on `localhost:10000` to reach your Scalingo database. (You'll still need to authenticate to the
 database with the credential you can get from the connection string)
 
+### Access a Database on Multiple Applications
+
+If you need to share a database addon between multiple applications, you can do it by adding an environment variable in every application wanting to communicate with the database.
+
+In the application containing the database, you should have an environment variable containing the connection string needed to connect to the database. For a PostgreSQL addon, it should look like `SCALINGO_POSTGRESQL_URL`. It contains your database address, as well as its credentials. Keep this value secret!
+
+Copy this value, and in the application that needs to access it, you can create a new environment variable, and give it the value you just copied. After a restart, your app will have the connection address it needs.
```

Comment:
```suggestion
Copy this value, and in the application that needs to access it, create a new environment variable, and set it to the value you just copied. After a restart, your app will have the connection address it needs.
```

Reply:
I think it's a bit more complex: the duplicated environment var probably has to be named in a specific way (if I name it `MY_DB`, the app/framework probably won't pick it up).
Also some frameworks/apps don't use the whole URL has connection string :-/

Reply:
I took your suggestion, but in any ways, when creating a new database on Scalingo: the environment variable is always the same. It's up to the user to adapt it.